worker/uniter/runner/jujuc/status-set.go

```diff
@@ -31,12 +31,100 @@ func (c *StatusSetCommand) Info() *cmd.Info {
 Sets the workload status of the charm. Message is optional.
 The "last updated" attribute of the status is set, even if the
 status and message are the same as what's already set.
+
+Further details:
+status-set changes what is displayed in juju status.
+status-set allows charms to describe their current status.
+This places the responsibility on the charm to know its status,
+and set it accordingly using the status-set hook tool.
+Changes made via status-set are applied without waiting for a
+hook execution to end and are not rolled back if a hook
+execution fails.
+
+The leader unit is responsible for setting the overall status
+of the application by using the --application option.
+
+This hook tool takes 2 arguments. The first is the status code
+and the second is a message to report to the user.
+
+Valid status codes are:
+    maintenance (the unit is not currently providing a service,
+	  but expects to be soon, e.g. when first installing)
+    blocked (the unit cannot continue without user input)
+    waiting (the unit itself is not in error and requires no
+	  intervention, but it is not currently in service as it
+	  depends on some external factor, e.g. an application to
+	  which it is related is not running)
+    active (This unit believes it is correctly offering all
+	  the services it is primarily installed to provide)
+
+For more extensive explanations of these status codes, please see
+the status reference page.
```

Comment:
what to do about links

Reply:
You can put the full link to that page for now: https://juju.is/docs/juju/status . The page is published so that should be all right. (Still not ideal but we'll take a fine comb to this whole thing later.)